src/renderer/patches/screenShareAudio.ts

```diff
@@ -0,0 +1,39 @@
+const original = navigator.mediaDevices.getDisplayMedia;
```

Comment:
make sure this only runs on the target platform and doesnt error if virtmic isnt active

Reply:
Should work now i believe?